recipes/bioconductor-a4base/meta.yaml

```diff
@@ -10,35 +10,35 @@ source:
     - 'http://bioconductor.org/packages/{{ bioc }}/bioc/src/contrib/{{ name }}_{{ version }}.tar.gz'
     - 'https://bioarchive.galaxyproject.org/{{ name }}_{{ version }}.tar.gz'
     - 'https://depot.galaxyproject.org/software/bioconductor-{{ name|lower }}/bioconductor-{{ name|lower }}_{{ version }}_src_all.tar.gz'
-  sha256: 5a6416fc098d10ad13114b608c251978eef4a43912f35079cb8b1d96ead6404e
+  sha256: ed117190a960d4e69725cf5fd1f2c7c785a5258c06eb897d80f0a57626ae7520
 build:
-  number: 1
+  number: 0
   rpaths:
     - lib/R/lib/
     - lib/
 requirements:
   host:
-    - bioconductor-a4core
-    - bioconductor-a4preproc
-    - bioconductor-annaffy
-    - bioconductor-annotationdbi
-    - bioconductor-biobase
-    - bioconductor-genefilter
-    - bioconductor-limma
-    - bioconductor-multtest
+    - 'bioconductor-a4core >=1.28.0,<1.30.0'
+    - 'bioconductor-a4preproc >=1.28.0,<1.30.0'
+    - 'bioconductor-annaffy >=1.52.0,<1.54.0'
+    - 'bioconductor-annotationdbi >=1.42.1,<1.44.0'
+    - 'bioconductor-biobase >=2.40.0,<2.42.0'
+    - 'bioconductor-genefilter >=1.62.0,<1.64.0'
+    - 'bioconductor-limma >=3.36.5,<3.38.0'
```

Comment:
In theory the this should probably `>=3.36.0`, in practice, it does not matter I guess.

Reply:
Yeah, that can wait for the skeleton to get updated.